### PR TITLE
Feature/RFE-748/Add translations

### PR DESCRIPTION
**Pull request by jsconan**
### Summary

Update the list of translations

### CODE REVIEW

File: locales/ar-arb/messages.po [hunk 1]
```diff
@@ -705,6 +741,7 @@ msgstr "إدارة مجموعات المواد"
 msgid "Manage test properties"
 msgstr "إدارة خصائص الرائز"
 
+#, tao-public
```
Comment (shpran) [inline locales/ar-arb/messages.po:744]: Do we need this comment?

File: locales/ar-arb/messages.po [hunk 2]
```diff
@@ -766,10 +803,18 @@ msgid "minutes"
 msgstr "دقائق"
 
 msgid "Missed required parameter 'id\\"
-msgstr "تم فقدان المعلم المطلوب 'id \\"
+msgstr "تم فقدان المعلم المطلوب 'id \\"\n"
+"\n"
+"msgid "
```
Comment (shpran) [inline locales/ar-arb/messages.po:808]: I think we have an issue here

File: locales/ar-arb/messages.po [hunk 3]
```diff
@@ -1380,6 +1459,14 @@ msgstr "عنوان القسم."
 msgid "The submission mode determines when the candidate's responses are submitted for response processing. A testPart in individual mode requires the candidate to submit their responses on an item-by-item basis. In simultaneous mode the candidate's responses are all submitted together at the end of the testPart."
 msgstr "يحدد وضع الإرسال وقت إرسال أجوبة المرشح لمعالجتها. يتطلب قسم من الرائز في الوضع الفردي من المرشح إرسال أجوبته على أساس كل بند على حدة. في الوضع المتزامن ، يتم تقديم جميع أجوبة المرشح معًا في نهايةقسم من الرائز.."
 
+msgid "The test cannot be saved because it currently contains invalid settings.\n"
+""
+msgstr ""
```
Comment (shpran) [inline locales/ar-arb/messages.po:1464]: Looks strange 🤔

File: locales/ar-arb/messages.po [hunk 4]
```diff
@@ -1380,6 +1459,14 @@ msgstr "عنوان القسم."
...
+
+msgid "The test cannot be saved because it currently contains invalid settings.\n"
+"Please fix the invalid settings and try again."
```
Comment (shpran) [inline locales/ar-arb/messages.po:1467]: Do we support multi-line syntax?

File: locales/da-DK/messages.po [hunk 5]
```diff
@@ -762,12 +804,18 @@ msgstr ""
 msgid "minutes"
 msgstr ""
 
+msgid "Missed required parameter \'id\"
```
Comment (shpran) [inline locales/da-DK/messages.po:807]: Looks like we have an issue here

File: locales/da-DK/messages.po [hunk 6]
```diff
@@ -1378,6 +1459,14 @@ msgstr ""
 msgid "The submission mode determines when the candidate's responses are submitted for response processing. A testPart in individual mode requires the candidate to submit their responses on an item-by-item basis. In simultaneous mode the candidate's responses are all submitted together at the end of the testPart."
 msgstr ""
 
+msgid "The test cannot be saved because it currently contains invalid settings.\n"
+""
+msgstr ""
+
+msgid "The test cannot be saved because it currently contains invalid settings.\n"
+"Please fix the invalid settings and try again."
```
Comment (shpran) [inline locales/da-DK/messages.po:1467]: Multi-line?

File: locales/ar-arb/messages.po (hunk 1 above)
Comment (jsconan) [reply locales/ar-arb/messages.po:744]: No idea, it was auto-generated

**Review by shpran — APPROVED** [review]
Looks ok 👍🏻

**Review by gabrielfs7 — APPROVED** [review]
LGTM

Cannot check it locally now as my env is crashed, but file seems straight forward and only sorting and a few fix added apart of the new strings

**Pull request merged**